objcgen/objcgenerator.cs

```diff
@@ -151,15 +151,17 @@ protected override void Generate (Type t)
 			var native_name = GetTypeName (t);
 			headers.WriteLine ();
 			headers.WriteLine ($"// {t.AssemblyQualifiedName}");
-			headers.WriteLine ($"@interface {native_name} : {GetTypeName (t.BaseType)}");
```

Comment:
what's the exact build that failed ? Apple has been removing this pattern from headers for a while - but maybe I never noticed they did not touch the macOS 32bits ones ?